wagtail_localize/views/edit_translation.py

```diff
@@ -1063,10 +1075,55 @@ def get_translation_progress(segment, locale):
                 cls=DjangoJSONEncoder,
             ),
             "has_editor_css": WAGTAIL_VERSION <= (4, 0),
+            "uses_legacy_header": uses_legacy_header
```

Comment:
`uses_legacy_header` and `has_editor_css` are pretty much the same in their use.

suggestion: change `uses_legacy_header` to `has_legacy_styling`, drop `has_editor_css` (replacing it with `has_legacy_styling`

Reply:
Fixed